### ISSUE TITLE
Error TypeError: logger.log is not a function

### ISSUE THREAD

**Issue by zvikarp** [ampproject/amphtml#30527]
## What's the issue?

In the latest version, I get the following error when running the validation.

```
TypeError: logger.log is not a function
    at DestroyableTransform.formatResults [as _flush] (C:\path-to-project\node_modules\gulp-amphtml-validator\index.js:97:12)
    at DestroyableTransform.prefinish (C:\path-to-project\node_modules\readable-stream\lib\_stream_transform.js:138:10)
    at DestroyableTransform.emit (events.js:315:20)
    at DestroyableTransform.EventEmitter.emit (domain.js:506:15)
    at prefinish (C:path-to-project\node_modules\readable-stream\lib\_stream_writable.js:619:14)    
    at finishMaybe (C:\path-to-project\node_modules\readable-stream\lib\_stream_writable.js:627:5)   
    at endWritable (C:\path-to-project\node_modules\readable-stream\lib\_stream_writable.js:638:3)   
    at DestroyableTransform.Writable.end (C:\path-to-project\node_modules\readable-stream\lib\_stream_writable.js:594:41)
    at DestroyableTransform.onend (C:\path-to-project\node_modules\readable-stream\lib\_stream_readable.js:577:10)
    at Object.onceWrapper (events.js:421:28)
```

It seems that fancy-log is misused, should be `logger.info` instead of `logger.log` https://github.com/ampproject/amphtml/blob/64c46649d55d5fb56e548455981de742af56a3a4/validator/js/gulpjs/index.js#L97

## How do we reproduce the issue?

create a new project with the latest packages to date (06/10/20), node version `v12.18.3` here is my `package.json`:

```
{
  "name": "projectName",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@ampproject/toolbox-optimizer": "^2.6.0",
    "browser-sync": "^2.26.12",
    "del": "^6.0.0",
    "gulp": "^4.0.2",
    "gulp-amphtml-validator": "^1.0.5",
    "gulp-autoprefixer": "^7.0.1",
    "gulp-cssnano": "^2.1.3",
    "gulp-file-include": "^2.2.2",
    "gulp-filter": "^6.0.0",
    "gulp-inject-string": "^1.1.2",
    "gulp-merge-media-queries": "^0.2.1",
    "gulp-noop": "^1.0.1",
    "gulp-plumber": "^1.2.1",
    "gulp-replace": "^1.0.0",
    "gulp-sass": "^4.1.0",
    "through2": "^4.0.2"
  }
}
```

run the gulp task:

```
gulp.task("validate", function validate() {
  return gulp
    .src(paths.html.dest + "/**/*.html")
    .pipe(gulpAmpValidator.validate())
    .pipe(gulpAmpValidator.format())
    .pipe(gulpAmpValidator.failAfterError());
});
```

## What browsers are affected?

Not browser-specific.

## Which AMP version is affected?

bug in npm package `gulp-amphtml-validator` latest version (`1.0.5`).

**Comment by honeybadgerdontcare** [comment]
This has been released as [1.0.6](https://www.npmjs.com/package/gulp-amphtml-validator/v/1.0.6)

**Comment by honeybadgerdontcare** [comment]
@zvikarp thank you for reporting this and identifying the root cause! please let us know if you encounter additional issues.
